packages/node_modules/@ciscospark/internal-plugin-team/test/integration/spec/mercury.js

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '@ciscospark/internal-plugin-wdm';
+import '@ciscospark/internal-plugin-team';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import {expectActivity} from '@ciscospark/test-helper-mocha';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+describe('plugin-team', () => {
+  describe('Team', () => {
+    let kirk, spock;
+
+    before(() => testUsers.create({count: 2})
+      .then((users) => {
+        [kirk, spock] = users;
+
+        kirk.spark = new CiscoSpark({
+          credentials: {
+            authorization: kirk.token
+          },
+          config: {
+            conversation: {
+              keepEncryptedProperties: true
+            }
+          }
+        });
+
+        spock.spark = new CiscoSpark({
+          credentials: {
+            authorization: spock.token
+          },
+          config: {
+            conversation: {
+              keepEncryptedProperties: true
+            }
+          }
+        });
+
+        return Promise.all([
+          kirk.spark.internal.mercury.connect(),
+          spock.spark.internal.mercury.connect()
+        ]);
+      }));
+
+    after(() => Promise.all([
+      kirk && kirk.spark.internal.mercury.disconnect(),
+      spock && spock.spark.internal.mercury.disconnect()
+    ]));
+
+    describe('when mercury event is received', () => {
+      let teamConvo, team;
+      before(() => kirk.spark.internal.team.create({
+        displayName: `team-${uuid.v4()}`,
+        participants: [
+          kirk,
+          spock
+        ]
+      })
+        .then((t) => {
+          team = t;
+        })
+        .then(() => kirk.spark.internal.team.createConversation(team, {
+          displayName: `team-conversation-${uuid.v4()}`,
+          participants: [
+            kirk
+          ]
+        }))
+        .then((c) => {
+          teamConvo = c;
+        }));
+
+      it('updates the displayName of an unjoined team convo', () => {
+        const update = {
+          displayName: `updated-team-convo--${uuid.v4()}`,
+          objectType: 'conversation'
+        };
+
+        const activityChecker = (activity) => activity.verb === 'update' && activity.object && activity.object.objectType === 'teamRoomStatus';
```

Comment:
try using `get()` from lodash to make the `activity.object.objectType...` bits easier to read